packages/fxa-auth-server/bin/update_firestore_from_oauth.js

```diff
@@ -21,6 +21,7 @@ const log = require('../lib/log')(
 );
 
 const CLIENT_ID = process.env.CLIENT_ID;
```

Comment:
Can we add an assertion here that the client has the proper case. Abort on `/[A-Z]/.test(CLIENT_ID)`?

Reply:
I can't right now, or you won't be able to fix the ones with the error.